jetty-server/src/main/java/org/eclipse/jetty/server/AllowedResourceAliasChecker.java

```diff
@@ -198,13 +221,43 @@ protected Path getPath(Resource resource)
         }
     }
 
+    private class AllowedResourceAliasCheckListener implements LifeCycle.Listener
```

Comment:
geeeze we need to add default methods on that listener interface!

Reply:
Opened as PR #7124